x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -184,6 +186,38 @@ export interface HostResultList {
   policy_info?: HostInfo['policy_info'];
 }
 
+// TODO update types when actual API is ready
```

Comment:
so this is based on what I saw with `localhost:9200/.fleet-actions/_search` but will change later.